test/src/test/resources/fixtures/chained transforms/ari-use-case/ari-argument-transform-on-renamed-field.yml

```diff
@@ -1,58 +0,0 @@
-name: ari argument transform on renamed field
```

Comment:
Duplicated test also exists at

```
fixtures/chained transforms/ari use case/ari-argument-transform-on-renamed-field.yml
```

Whereas this file is at

```
fixtures/chained transforms/ari-use-case/ari-argument-transform-on-renamed-field.yml
```

Not sure how it happened…